arc/src/main.c

```diff
@@ -762,55 +768,132 @@ static int stop_gyro_trigger(struct device *dev)
     return 0;
 }
 
+#ifdef BUILD_MODULE_SENSOR_LIGHT
+static double cube_root_recursive(double num, double low, double high) {
```

Comment:
> Personally I think we should be returning only raw data from sensors

That would be a violation of the specification that this purports to implement. The [AmbientLightSensor specification](https://www.w3.org/TR/ambient-light/) defines an `illuminance` property on the object located at `event.reading`, eg.

```js
let sensor = new AmbientLightSensor();
sensor.start();
    
sensor.onchange = function(event) {
  console.log(event.reading.illuminance);
};
```


`illuminance` is defined as ambient light level in lux.